apps/src/blockly/addons/cdoFieldDropdown.js

```diff
@@ -95,10 +85,10 @@ export default class CdoFieldDropdown extends GoogleBlockly.FieldDropdown {
    * @override
    */
   toXml(element) {
-    super.toXml(element);
     if (this.config) {
       element.setAttribute('config', this.config);
     }
+    super.toXml(element);
```

Comment:
Can you explain why we wanted to move this down?

Reply:
Sure! 
Short answer: to be consistent with `fromXml`.
Long answer: `super.toXml` assigns to the element (`textContent`) the value of the field. We override to support the `config` attribute if it exists and the placement of the call to `super` doesn't matter in this function.
But the order does matter in `fromXml` above. The call to `super.fromXml` must happen after we check the `config`. So I moved this line just to be consistent with `fromXml`.

Reply:
Perfect, thanks!